plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/SubversionExtension.java

```diff
@@ -199,8 +199,7 @@ public SubversionExtension(final ActionManager actionManager,
 
         contextGroup.getTemplatePresentation().setSVGResource(resources.svn());
 
-        DefaultActionGroup mainContextMenuGroup = (DefaultActionGroup)actionManager.getAction(IdeActions.GROUP_MAIN_CONTEXT_MENU);
-        mainContextMenuGroup.addSeparator();
+        DefaultActionGroup mainContextMenuGroup = (DefaultActionGroup)actionManager.getAction("resourceOperation");
```

Comment:
This is inner part of main context menu (https://github.com/eclipse/che/blob/24bdc4d09f6435df9b51ea1a6de25bbdd1126f20/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/IdeActions.java#L34)
We haven't a constant for this section, so it is defined by string everywhere.